### DISCUSSION TITLE
We have more signatures and stars. The community supports RMS

### DISCUSSION THREAD

**Discussion by ghost** [rms-support-letter/rms-support-letter.github.io#4770] [General]
### open letter
stars: 575
fork: 2.1k
translated languages: 17
**signatures:** 2895

### support letter
stars: 1.9k
fork: 3.8k
translated languages: 26
**signatures:** 4080

The community has spoken. Majority of us want RMS to stay. You don't need to agree with RMS, because this issue is much greater than RMS, they target the free software movement and our freedom of expression. I signed this letter, even though I disagree with RMS on many issues.

**Comment by 6r1d** [comment]
> they target the free software movement and our freedom of expression

Some time ago, I'd tell you that you are exaggerating. Now, though, with evidence like "[Open is cancelled](https://medium.com/@beewithablog/open-is-cancelled-da7dd6f2aaaf)", I sadly have to agree: you are completely right there.

**Comment by KOLANICH** [comment]
IT’S *NEO*LIBERALS ON THE MARCH GET IN THE CAR

**Comment by KOLANICH** [comment]
They have stopped to accept signatures via PRs (is it an effort to make comparison more difficult?), also count of forks is not an approximate of a count of PRs, since some people don't understand difference between a "star" and a "fork" and create unneeded forks and since many prefer to delete forks after their prs are merged, and count of stars is not count of signatures.

**Comment by jobbautista9** [comment]
> IT’S _NEO_LIBERALS ON THE MARCH GET IN THE CAR

Funnily enough, true neoliberals won't care about this. After all they are against populism.

**Comment by KOLANICH** [comment]
I don't think that there is such a thing as a true `neoliberal`, because `neoliberal`s are in fact not liberal and the word `neoliberal` is just a purportedly misleading euphemism for a `fascist`, which has falled out of favour after WW2.

**Comment by ghost** [comment]
neoliberal, liberal, libertarian - all 3 are different

**Comment by teunissenstefan** [comment]
@ItsukiNakano

"Russians"
This is quite obviously 100% accurate. Nevermind the fact that you joined 15 minutes ago.

**Comment by jobbautista9** [comment]
Aba aba, I didn't know that Filipinos are Russian. Anyone wants to join me in establishing the Philippine Soviet Socialist Republic?

**Comment by nukeop** [comment]
bye bye

**Comment by KOLANICH** [comment]
@ItsukiNakano, How large is the share of votes for `rms-open-letter` that was collected by mail? Can `rms-open-letter` folks proove the votes were by real persons and not forged by the ones collecting them? I require them to publish all the mails they registered as votes with their DKIM signatures and proofs that the addresses belong to real developers with well-known recognized contributions to open-source.

**Comment by KOLANICH** [comment]
@jobbautista9, don't you know that Etrusc ("этруски") means "it's Russians" ("это - русские") in Russian ? Russians are everywhere, it just they don't always know about it ;)

**Comment by nukeop** [comment]
> @ItsukiNakano, How large is the share of votes for `rms-open-letter` that was collected by mail? Can `rms-open-letter` folks proove the votes were by real persons and not forged by the ones collecting them? I require them to publish all the mails they registered as votes with their DKIM signatures and proofs that the mails belong to real developers with well-known recognized contributions to open-source.

Our process is 100% transparent. Everything is public, down to email headers.

**Comment by KOLANICH** [comment]
@nukeop, what was the deleted comment about? I haven't received it by email.

**Comment by teunissenstefan** [comment]
@KOLANICH  I forgot what he said but he attached [this image](https://user-images.githubusercontent.com/81519284/112754631-a137bf00-8fe5-11eb-9fb2-5eba0d71b196.png)

**Comment by KOLANICH** [comment]
@teunissenstefan, I saw the comment with the image, but I am not sure that that comment was deleted by a moderator. The one with the pic has disappeared too (and I see absolutely no point in censoring it), but the marking about comment deletion by a moderator is after the messages in which a comment with a pic was mentioned, so I feel like there may be a yet another comment, that was in place of the marking, before it was deleted. Though I can misunderstand how GH comment deletion by a moderator works.

**Comment by nukeop** [comment]
It was pointless spam I don't want to see here

**Comment by shenlebantongying** [comment]
@nukeop maybe hide or fold better than delete?

**Comment by nukeop** [comment]
There's no value in keeping it here imo.

**Comment by ghost** [comment]
I made a website that shows how many more signatures rms-support-letter has than rms-open-letter. https://rms-letters.spheeresx.repl.co/

**Comment by shenlebantongying** [comment]
@SpheeresX I think maybe `2812+48+16` is much more clear than `2860`. 48 -> (orgs) 16 ->  (initial signers)

**Comment by ghost** [comment]
@shenlebantongying I'll try to add that tomorrow, thanks!

**Comment by srid** [comment]
> this issue is much greater than RMS

Here are a couple of links for folks trying to comprehend what this means:

- https://reclaimthenet.org/
- https://newdiscourses.com/tftw-critical-race-theory/

**Comment by jobbautista9** [comment]
By the way, this seems something that should be converted into a discussion, since there's no issue here at all. Any objections?

**Comment by KOLANICH** [comment]
I feel like it is not beneficial for this project and just introduces 2 places to look at. In fact I don't even know any real use-cases for them issues cannot handle, except just creating a dedicated place for offtop threads.

**Comment by KOLANICH** [comment]
Discussions threads look and feel like a piece of shit. Also this shit doesn't really work in old browsers (I refuse to discuss about the reasons to use old browsers and other old and vulnerable shit. You already know which orgs and which devs to blame for (even though we cannot blame persons and corporations for pursuing their profits)). Could this be converted back into an issue?

**Reply by Bluesboy** [reply]
C'mon man issues have purpose.

**Comment by KOLANICH** [comment]
I wonder if it is acceptable to some well-known contributors of some well-known orgs that have not yet signed the anti-Stallman letter on behalf of the org to block users of GH who have signed pro-Stallman letter using the tools like https://github.com/travisbrown/octocrabby (that may cause them being unable to comment the issues and PRs created by them, and if he is the owner of the repo, can just expell users from a repo)? Won't it be considered abuse of power? I guess we may need to detect such a behavior.

1. a script scrapes all the signatories of the harrass letter
2. the script finds some issues by users of the harrass letter using GH search `is: issue user: name`
3. the script samples some issues in different repis of different authors and opens them. Then checks for the DOM node with `You can't perform this action at this time.`. 
4. How about petition GH to alter the block feature in the way making it unusable for harrassment purposes?

**Reply by Bluesboy** [reply]
We have 
https://github.com/rms-support-letter/rms-support-letter.github.io/pull/4085
and
https://github.com/rms-support-letter/rms-support-letter.github.io/pull/2535
IMO it's more than 48 corporative scum on the other side
